contracts/src/BeefyClient.sol

```diff
@@ -289,6 +302,8 @@ contract BeefyClient {
             prevRandao: 0,
             bitfieldHash: keccak256(abi.encodePacked(bitfield))
         });
+
+        emit NewTicket(msg.sender, commitment.blockNumber);
```

Comment:
We should add the `ticketId` to `NewTicket` and `NewMMRRoot` events to be able to link them.